etna/models/nn/deepar_new/deepar.py

```diff
@@ -104,13 +117,19 @@ def forward(self, x: DeepARBatchNew, *args, **kwargs):  # type: ignore
         decoder_real = x["decoder_real"].float()  # (batch_size, decoder_length, input_size)
         decoder_target = x["decoder_target"].float()  # (batch_size, decoder_length, 1)
         decoder_length = decoder_real.shape[1]
-        weights = x["weight"]
+        weights = x["weight"]  # (batch_size)
+        segment_ids = x["segment_idx"]  # (batch_size)
+        encoder_embeddings = (
```

Comment:
Don't get the purpose, you need to get embeddings for unique segment_ids and then just repeat it for encoder/decoder points